OneSignalExample/Assets/OneSignal/Editor/OneSignalEditorScript_Android.cs

```diff
@@ -43,7 +43,7 @@ public class OneSignalEditorScriptAndroid : AssetPostprocessor {
    // Copies `AndroidManifestTemplate.xml` to `AndroidManifest.xml`
    //   then replace `${manifestApplicationId}` with current packagename in the Unity settings.
    private static void createOneSignalAndroidManifest() {
-      string oneSignalConfigPath = "Assets/Plugins/Android/OneSignalConfig/";
+      string oneSignalConfigPath = Application.dataPath + "/Plugins/Android/OneSignalConfig/";
```

Comment:
This may work but it isn't clear from the Unity docs if `Application.dataPath` is the same or different if running the Editor on Windows vs macOS. Also it looks like it may return a full path instead of a relative path which will needs to be tested with `File.Copy` below.
Lastly it doesn't seem the Assets folder can be configurable. If it is then we also have to consider where the `OneSignalSDK.unitypackage` extracts the template file.
https://docs.unity3d.com/ScriptReference/Application-dataPath.html

Reply:
Check your latest update, it is works without code fix.
Thanks!